templates/frontpage_login.mustache

```diff
@@ -41,7 +41,7 @@
         </div>
     {{/output.has_identity_providers}}
 
-    <p><a href="{{{ config.wwwroot }}}/login/forgot_password.php">{{#str}}forgotten{{/str}}</a></p>
+    <p><a href="{{{ config.wwwroot }}}/login/forgot_password.php">Already have your account? <!-- {{#str}}forgotten{{/str}} --></a></p>
```

Comment:
Replace the 'forgotten' value in the language files.

Reply:
See above for sample code